components/crashtest/android/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Newb question: I've seen this file in other components and noticed that it looks like all comments. Out of curiosity is it required? Are we adding this file in order to allow for potential configuration?

Reply:
That is a great question, and I don't have a good answer TBH. I think it's a convention we inherited from android-components but you're right, they don't seem to actually *do* anything.